### DISCUSSION TITLE
The starting point of the Equatorial coordinate system.

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#2009] [General]
I am interested in using the Equatorial coordinate system (RA/ DEC (on date)), But that the starting point will not be at the beginning of Aries, but at the point where the zodiac intersects the equator.
(I'm researching a book by a medieval astronomer, and I think he used such a system.)
Alternatively, where in the stellarium can be found the exact point where the zodiac crosses the equator at a given time.
Thank you all.

**Comment by gzotti** [comment]
The starting point of the ecliptic is defined as the point where it intersects the equator and where the sun crosses northwards. This is called "Beginning of the Sign of Aries", or "First Point of Aries", independent from the visible stars of the constellation of Aries. Ecliptical coordinates are given in 0...360° from this point today. In earlier times, it would have been given in terms of Sign+[0...30°], like "Taurus 20" for &lambda;=50°.  

The intersection between equator of date and ecliptic of date is visible on the screen if you activate the respective lines and markers.

**Comment by tzmosh** [comment]
Thank you.
I also thank you in general for Stellarium...
